src/DynamoCoreWpf/ViewModels/Core/AnnotationViewModel.cs

```diff
@@ -254,6 +255,7 @@ public bool IsExpanded
                 WorkspaceViewModel.HasUnsavedChanges = true;
                 AddGroupToGroupCommand.RaiseCanExecuteChanged();
                 RaisePropertyChanged(nameof(IsExpanded));
+                AnnotationModel.UpdateErrorAndWarningIconVisibility();
```

Comment:
Might make sense to move this call to the AnnotaionModel?
https://github.com/DynamoDS/Dynamo/blob/master/src/DynamoCore/Graph/Annotations/AnnotationModel.cs#L327

Reply:
Makes sense, have removed all references from AnnotationViewModel and set to Private.